content/working_with/manager/using-secrets.md

```diff
@@ -18,6 +18,11 @@ All the values of the secrets are encrypted in the database. When you create a s
 A secret with a hidden value means the value is only shown to the user who created it, tenant managers and sys-admins.
 Users can use the secret according to the user roles and the visibility of the secret.
 
+### Secrets from remote provider
+
+Secret value can be provided by remote "Secrets Provider", like Vault or another Cloudify Manager.
```

Comment:
don't say `Cloudify Manager` but rather use the macros, `{{< param product_name >}}` = `Cloudify`,  `{{< param cfy_manager_name >}}` = `Cloudify Manager`

This is for whitelabeling, where some customers relabel the manager with their own name

Reply:
(btw, the available params are listed in config.toml)